lib/shoot/version.rb

```diff
@@ -1,3 +1,3 @@
 module Shoot
-  VERSION = "0.2.0"
+  VERSION = "0.2.2"
```

Comment:
Next PRs please do not change the version, I have to publish the gem anyway, so I think it's not a good workflow to do it, cause it will cause conflicts.